README.md

```diff
@@ -192,6 +191,11 @@ Replace `NAMESPACE` with the namespace that the operator is running in.
 To remove more than one operator, repeat these steps for each operator namespace.
 
 
+# Recent Changes
+
+See [Recent changes](site/recent-changes.md) for recent changes to the weblogic operator, including any backwards incompatible changes.
```

Comment:
Use "WebLogic" anytime you mention the product.  However, here, you can just say "the operator."

Reply:
OK